ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -842,4 +846,25 @@ sub relativize_url {
     return $rel_path;
 }
 
+# Remove any level of indentation (spaces or tabs) from each code block consistently
+# Adapted from: https://metacpan.org/source/HAARG/MetaCPAN-Pod-XHTML-0.002001/lib/Pod/Simple/Role/StripVerbatimIndent.pm
+sub trim_leading_whitespace {
+    my ($para) = @_;
+
+    # Start by converting tabs to spaces
+	$tabstop = 4;
```

Comment:
8 is the only acceptable value for tab width in this context.  Perl's own pod will not render correctly with any other tab width.